amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpConnectionProvider.scala

```diff
@@ -67,8 +67,8 @@ final class AmqpDetailsConnectionProvider private (
     val handshakeTimeout: Option[Int] = None,
     val shutdownTimeout: Option[Int] = None,
     val networkRecoveryInterval: Option[Int] = None,
-    val automaticRecoveryEnabled: Option[Boolean] = None,
-    val topologyRecoveryEnabled: Option[Boolean] = None,
+    val automaticRecoveryEnabled: Option[Boolean] = Some(false),
+    val topologyRecoveryEnabled: Option[Boolean] = Some(false),
```

Comment:
Shouldn't the Options be dropped in favour of flat Booleans?

Reply:
I left the Option just for consistency since all the other parameters are optional.
But I have no strong opinion on the matter.

Should I change them to plain Boolean?

Reply:
Yes, please.